docs/site-generator/makefile-templates/project.yml

```diff
@@ -6,3 +6,4 @@ repoVersionUrl: https://github.com/${OWNER}/${REPOSITORY}/releases/tag/${VERSION
 issuesUrl: https://github.com/${OWNER}/${REPOSITORY}/issues
 docUrl: https://godoc.org/gopkg.in/${OWNER}/${REPOSITORY}.${VERSION}
 baseURL: ${BASEURL}
+languages: ${LANGUAGES}
```

Comment:
in a much more dirty way; with the new approach the `Makefile` is much more easy to read, and the `aside.html` too.